bundles/automation/org.eclipse.smarthome.automation.rest/META-INF/MANIFEST.MF

```diff
@@ -17,5 +17,6 @@ Import-Package:
  org.eclipse.smarthome.config.core,
  org.eclipse.smarthome.core.common.registry,
  org.eclipse.smarthome.io.rest,
+ org.osgi.service.component.annotations,

```

Comment:
IIRC there has been a commit some time ago that removes all that imports.
Annotations does not need to be present at runtime (by the JVM) and the one here are known to be used at build time only.